SPEC.md

```diff
@@ -64,50 +64,55 @@ It includes a *rootfs* with all of the files that will exist in the root of the
 
 ### Image Archives
 
-The ACI archive format aims for flexibility and relies on very boring technologies: HTTP, gpg, tar and gzip.
-This set of formats makes it easy to build, host and secure a container using technologies that are battle-tested.
+The ACI file format ("image archive") aims for flexibility and relies on standard and common technologies: HTTP, gpg, tar and gzip.
+This set of formats makes it easy to build, host and secure a container using technologies that are widely available and battle-tested.
 
-Image archives MUST be a tar formatted file.
-The image may be optionally compressed with gzip, bzip2, or xz.
-After compression, images may also be encrypted with AES symmetric encryption.
+- Image archives MUST be a tar formatted file.
+- All files in the image MUST maintain all of their original properties, including timestamps, Unix modes, and extended attributes (xattrs).
+- Image archives MUST be named with the suffix `.aci`, irrespective of compression/encryption (see below).
+- Image archives SHOULD be signed using PGP, in detached signature mode.
+- Image signatures MUST be named with the suffix `.sig`.
 
+There are two further transformations that may be applied to image archives for transport:
+- Image archives MAY be compressed with `gzip`, `bzip2`, or `xz`.
```

Comment:
@vbatts It makes discovery of the URLs harder and doesn't add much value since we can detect the compression via header snooping.